sympy/printing/pycode.py

```diff
@@ -571,8 +571,6 @@ def _print_Symbol(self, expr):
                        'reserved keyword in this language.')
                 raise ValueError(msg.format(name))
             return name + self._settings['reserved_word_suffix']
-        elif '{' in name:   # Remove curly braces from subscripted variables
```

Comment:
It should be possible to delete this method completely now. (And maybe some other identical ones if there are any.)

Reply:
I mean: you should remove the whole `_print_Symbol`-method here as it will be inherited from `codeprinter` now.

Reply:
I see, thanks！